src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Quadrangle {
+    public IsoscelesTrapezoid(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double area() {
+        return super.area();
```

Comment:
The method 'area' should not simply call 'super.area()'. Each figure should have its own way to calculate the area. Please implement the area calculation for an isosceles trapezoid based on its specific properties.